helpers.sh

```diff
@@ -353,7 +364,16 @@ git_log() {
 }
 
 
-export_coveralls_data() {
+prepare_coverage_data() {
+
+  print_info "Preparing coverage templates..."
+  cp "${SMALLTALK_CI_HOME}"/coverage/* "${SMALLTALK_CI_BUILD}"
+  chmod +xw "${SMALLTALK_CI_BUILD}/upload-coverage.sh"
```

Comment:
The `chmod`s should not be necessary if they are executable (and writable in the first case) on your system. The permissions should be stored correctly in Git.

Reply:
That was my intuition but somehow the permissions got lost when running in the GitHub action, but can be just that I've run this several times until I got it working. Debugging things running on a remote server without any kind of interaction possible sucks :( . If you want to remove it, do it in an isolated commit so it can be easily recovered if don't work.